### DISCUSSION TITLE
Load a plugin after a lazy loaded plugin loads

### DISCUSSION THREAD

**Discussion by neilsabde** [wbthomason/packer.nvim#555] [General]
I'm not sure if you can do this but can I load a plugin after a lazy loaded plugin? For example, I have configured treesitter to load on BufRead and when I try to use nvim-ts-autotag, it says `module 'nvim-treesitter.configs' not found`. So is there a way I can load autotag after treesitter loads? I have tried using after = 'nvim-treesitter' but I still get the same error.

**Comment by wbthomason** [comment]
The `after` key should be the right way to do this - did you recompile/run sync after trying?

(Also sorry for the delay in responding; I've been away (#562))

**Reply by neilsabde** [reply]
Glad you're back :). Using the after keyword worked but I had to change the event in treesitter from 'BufRead' to 'BufWinEnter'.

**Reply by wbthomason** [reply]
Huh, that's strange. Does that behavior still replicate for you?

**Reply by neilsabde** [reply]
If treesitter loads on BufWinEnter everything works fine but if we load it on BufRead the treesitter plugins won't load even if the after keyword is set. Minimal config: https://gist.github.com/hackorum/013897121dc97b85edc26526477b4b0d
